Use the dbus-binding-tool from PATH, not the generated one.

RP - 12/1/09

Index: dbus-glib-0.76/tools/Makefile.am
===================================================================
--- dbus-glib-0.76.orig/tools/Makefile.am	2009-01-12 10:11:02.000000000 +0000
+++ dbus-glib-0.76/tools/Makefile.am	2009-01-12 10:12:01.000000000 +0000
@@ -3,8 +3,8 @@
 nodist_libdbus_glib_HEADERS = dbus-glib-bindings.h
 libdbus_glibdir = $(includedir)/dbus-1.0/dbus
 
-dbus-glib-bindings.h: dbus-bus-introspect.xml $(top_builddir)/dbus/dbus-binding-tool$(EXEEXT)
-	$(top_builddir)/dbus/dbus-binding-tool --mode=glib-client --prefix=dbus_bus --output=dbus-glib-bindings.h dbus-bus-introspect.xml
+dbus-glib-bindings.h: dbus-bus-introspect.xml
+	dbus-binding-tool --mode=glib-client --prefix=dbus_bus --output=dbus-glib-bindings.h dbus-bus-introspect.xml
 
 BUILT_SOURCES = dbus-glib-bindings.h dbus-bus-introspect.xml
 
